libs/journeys/ui/src/components/StepHeader/StepHeader.tsx

```diff
@@ -46,7 +48,7 @@ export function StepHeader({
           justifyContent="space-between"
           spacing={2}
           sx={{
-            px: { xs: 6, lg: 0 },
+            px: { xs: 4, lg: 0 },
```

Comment:
I ended up using 3 here to make sure the logo alignment was correct, will this work for you too?
```suggestion
            px: { xs: 3, lg: 0 },
```

Reply:
Yeah, I don't see any issues with that

Reply:
Great, can you make this change then?

Reply:
Yeah I forgot to add it.. my bad haha